### DISCUSSION TITLE
抓轨基础与常见误区

### DISCUSSION THREAD

**Discussion by giscus** [ZexWoo/Giscus#4] [Announcements]
# 抓轨基础与常见误区

玄学长久、持续地戕害着初探门径的发烧友，本文旨在揭露其胡说八道的本质。2019-11-04 是本文首次发表在某音乐论坛的日期。

https://zexwoo.blog/posts/knowledge/ripping-basic-and-common-misconceptions/

**Comment by candydre** [comment]
鬼斧神工119译的《声音的重现 第三版》今年出版了。

**Comment by VicaYang** [comment]
AccurateRip详解的链接挂了，应当是https://zexwoo.blog/posts/knowledge/accuraterip/

**Reply by ZexWoo** [reply]
确实，本文的链接都是旧的，打不开，我全都更新掉。

**Comment by liups233** [comment]
博主你好，我也是刚入门抓轨的小白，想问下用 EAC 抓取的速度是越低越好吗（不考虑时间成本）

我发现我的光驱（日立 LG WH16NS60）好像不能在 EAC 里设置速度，不管选择 4X 8X 还是更高，实际读取速度都不超过 2X；只有在选择速度为「当前的」，才能以更快的速度读取（不超过 6X）。所以我就好奇读取速度越低是不是越稳定，但又担心长时间读取会使光头过热

**Reply by liups233** [reply]
补充：我试过把「允许驱动器降速」这个选项关掉，但是速度仍然很慢。

**Reply by ZexWoo** [reply]
没有必要限制速度，光驱会在有需要的情况下降速。能顺利读取的话，让光驱按自己能力尽快跑就可以了。

**Comment by izumihiroaki** [comment]
我在一个log里看到Defeat audio cache显示no，请问这个是影响什么的，抓轨日志那边不能评论，发这边了，不好意思

**Reply by ZexWoo** [reply]
EAC 抓轨会对每个块读至少两遍并相互比较来确保无误（如果它感觉有问题，就会继续增加读取次数，不超过 82 次），如果允许缓存，后续读取就可能就来自先前读取留下的缓存，则两者 100% 一致，就失去了「多读几遍来确保无误」这个行为的意义，这就是我们要清空音频缓存的原因。

当然了，这样的设置实际上是比较粗暴的，没有考虑光驱自身属性，但这个无脑设置能保证你不会被缓存坑。

更多信息可以参考：https://wiki.hydrogenaudio.org/index.php?title=EAC_Drive_Options#Drive_caches_audio_data

> In order for secure mode to work properly, every read request made by EAC must cause the drive to seek data from the CD. If your drive caches audio, subsequent requests for the same data may result in the drive only fetching this data from its buffer, rather than from the physical disc. To prevent this from happening, EAC has a routine to ensure previously requested data gets flushed from drive's cache. This is done by having the drive read extra data from the disc—more data than the cache can store.
